pkgs/development/libraries/pipewire/default.nix

```diff
@@ -131,6 +131,9 @@ let
     ++ lib.optional pulseTunnelSupport libpulseaudio
     ++ lib.optional zeroconfSupport avahi;
 
+    # Optional check dependency.
+    checkInputs = lib.optional withValgrind valgrind;
```

Comment:
Valgrind headers are apparently vendored now so valgrind should not be needed on any platform:

https://gitlab.freedesktop.org/pipewire/pipewire/commit/d9cc1a25f179e43bd76b8f0bc80927a20de8ce55

Reply:
> Currently `valgrind` is only used in [an optional test](https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/0.3.40/test/meson.build#L109).